cms/djangoapps/contentstore/views/item.py

```diff
@@ -1247,6 +1248,12 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
     return xblock_info
 
 
+def highlights_setting():
+    namespace = u'dynamic_pacing'
+    switch = u'studio_course_update'
+    return WaffleSwitch(namespace, switch)
```

Comment:
Nit: I think this could just be a module level value, right?

Reply:
Yes, it can be.  We do declare these at the module level in other places.  The problem with this module is that it's way too huge and I didn't want to pollute it further - so I thought about encapsulating these constants within a localized function.

Reply:
That's fair. Could you have done just `highlights_setting = WaffleSwitch(u"dynamic_pacing", u"studio_course_update")`?

Reply:
Yes!  That would have been fine and even better!
The code evolved this way since I initially wrote it as a function that returned a boolean (`is_highlights_enabled`) - and then changed it to returning the switch object itself so it's available for test code.  So I must have kept it as a function when I modified it.  Thanks for pointing this out.